nixos/modules/services/misc/gitlab.nix

```diff
@@ -176,16 +178,15 @@ let
     MALLOC_ARENA_MAX = "2";
   } // cfg.extraEnv;
 
-  runtimeDeps = with pkgs; [
```

Comment:
You can do this in the gitlab package:
```
  # This 11 should be more or less in sync with https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab.rb#L36
  postPatch = ''
    echo '${lib.substring 0 11 data.rev}' > REVISION
  '';
```
and remove git from all runtime inputs except gitlab-config :smirk: